packages/cli/src/lib/snapshot/snapshot.ts

```diff
@@ -39,6 +39,20 @@ export class Snapshot {
     this.displayExpandedPreview();
   }
 
+  public hasChangedResources(): boolean {
+    throw new Error(
```

Comment:
When the TODO is processed, would that still be an error ? 🤔 

I think it should be still a "success" from the command/cli standpoint, just that nothing happens and we inform the end user.